admin/src/components/ContributionMessages/slots/ContributionMessagesListItem.vue

```diff
@@ -1,11 +1,24 @@
 <template>
   <div class="contribution-messages-list-item">
-    <div v-if="message.isModerator" class="text-right is-moderator">
+    <div
+      v-if="message.isModerator"
+      class="text-right is-moderator"
+      :class="message.type === 'MODERATOR' ? 'is-hidden-moderator-message p-2' : ''"
+    >
       <b-avatar square variant="warning"></b-avatar>
       <span class="ml-2 mr-2">{{ message.userFirstName }} {{ message.userLastName }}</span>
       <span class="ml-2">{{ $d(new Date(message.createdAt), 'short') }}</span>
-      <small class="ml-4 text-success">{{ $t('moderator') }}</small>
+      <small
+        class="ml-4 text-success"
+        :class="message.type === 'MODERATOR' ? 'text-success' : 'text-warning'"
+      >
+        {{ $t('moderator') }}
+      </small>
       <parse-message v-bind="message"></parse-message>
+      <small v-if="message.type === 'MODERATOR'">
+        <hr />
+        Diese Nachricht ist nur für die Moderatoren sichtbar!
```

Comment:
Localize. `This message can only be seen by moderators`

Reply:
done